lib/project_types/script/tasks/ensure_env.rb

```diff
@@ -0,0 +1,77 @@
+require "shopify_cli"
+
+module Script
+  module Tasks
+    class EnsureEnv < ShopifyCli::Task
```

Comment:
This class could easily be an application service, but I thought it might make sense to keep this pattern that the other project types use. Thoughts?

Reply:
I think since it's within Scripts, it should follow the patterns in here.  I agree, it does look like it belongs to the application layer.